zerver/lib/export.py

```diff
@@ -2116,15 +2120,12 @@ def _get_exported_s3_record(
         record["file_name"] = file_name
 
     if "user_profile_id" in record:
-        user_profile = get_user_profile_by_id(int(record["user_profile_id"]))
-        record["user_profile_email"] = user_profile.email
-
-        # Fix the record ids
         record["user_profile_id"] = int(record["user_profile_id"])
+        record["user_profile_email"] = user_id_to_email[record["user_profile_id"]]
```

Comment:
I wonder if we could just stop storing a (fake) email in the S3 metadata here; I don't think it's actually useful? Something to think about when doing our next migration to rewrite the S3 metadata.